hikari/impl/buckets.py

```diff
@@ -303,6 +327,28 @@ def drip(self) -> None:
         if not self.is_unknown:
             self.remaining -= 1
 
+    def resolve(self, real_bucket_hash: str) -> None:
```

Comment:
isn't this function unnecessary in it's current state?

Reply:
Kinda. It is meant to be an interface to updating the buckets name without doing it directly